{{cookiecutter.project_slug}}/fabfile.py

```diff
@@ -242,3 +242,21 @@ def get_media(directory=''):
             media_bucket=env.media_bucket, media=env.media, directory=directory
         )
     )
+
+
+@task
+@runs_once
+@roles('web')
+def get_env():
+    """ Get the current environment variables."""
+    run('set')
```

Comment:
Love the idea, although for me this outputs a fair amount of other stuff as well.

Thoughts on:

```
env.output_prefix = False
run('export')
```

Instead?

The even less alternative to export would be `printenv`, but that makes it less easy to copy/paste into postactivate files!